src/Microsoft.Performance.SDK/Processing/CustomDataProcessorWithSourceParser.cs

```diff
@@ -374,7 +374,7 @@ ISourceDataCookerFactory cookerFactory
 
         private void EnableInternalTables(IEnumerable<TableDescriptor> tables)
         {
-            foreach (var table in tables.Where(td => td.IsInternalTable))
+            foreach (var table in tables.Where(td => td.IsInternalTable && td.RequiresDataExtensions()))
```

Comment:
shouldn't this be not requires?

Reply:
nevermind.. I'm dumb :)

Reply:
hrm, how did this pass?

Reply:
oh, no
this is correct

Reply:
this is in the "WithSourceParser" class. I think you're thinking of the base class